queries/javascript/folds.scm

```diff
@@ -13,4 +13,7 @@
   (switch_statement)
   (switch_case)
   (import_statement)
+  (if_statement)
+  (try_statement)
```

Comment:
probably should add `catch_clause` as well to this.

Reply:
sure thing